packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.2.1]
+
+### Changed
+
+- Upgrade TypeScript to v5.0 and set `moduleResolution` option to `Node16` ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/base-controller` from `^6.0.0` to `^6.0.2` ([#4517](https://github.com/MetaMask/core/pull/4517), [#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/keyring-api` from `^8.0.0` to `^8.0.1` ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/keyring-controller` from `^17.1.0` to `^17.1.1` ([#4527](https://github.com/MetaMask/core/pull/4527))
```

Comment:
We are bumping this to 17.1.2 in this PR:

``` suggestion
- Bump `@metamask/keyring-controller` from `^17.1.0` to `^17.1.2` ([#4527](https://github.com/MetaMask/core/pull/4527), [#4548](https://github.com/MetaMask/core/pull/4548))

Reply:
```suggestion
```

- See https://github.com/MetaMask/core/pull/4556